### PR TITLE
Gov 303

### PR DESCRIPTION
**Pull request by tyler17**
Show exactly when a spell has passed (scheduled in pause) and been/can be executed.  Get this data from the blockchain.  No longer depend on the cms to know when a spell has been executed. Adding end_approvals to the cms for a proposal is now optional, the UI works with or without it.
Notes
-use the mocked cms to easily see what the tag looks like when a proposal is passed but not yet executed, and uncomment some lines in src/_mock/topics to see the executed state
-requires the sdk changes made in https://github.com/makerdao/dai.js/pull/208, which hasn’t been merged or published yet

### CODE REVIEW

File: src/pages/Timeline.js [hunk 1]
```diff
@@ -180,58 +181,59 @@ const Timeline = ({
...
+                <div style={{ display: 'flex' }}>
+                  <Tag mr="16" green>
+                    Governing Proposal
+                  </Tag>
+                  <Tag>{`Passed on ${formatDateWithTime(
+                    governingProposal.datePassed
+                  )}${
+                    governingProposal.end_approvals
+                      ? ' with ' +
+                        formatRound(governingProposal.end_approvals) +
```
Comment (jparklev) [inline src/pages/Timeline.js]: should this be bold?

Comment (tyler17) [reply src/pages/Timeline.js]: Yeah, but do you have any idea why the <b> tag wouldn't do anything? If I use <i> it italicizes, but <b> does nothing

Comment (jparklev) [reply src/pages/Timeline.js]: hmm interesting, my only guess would be that the `font-weight` being set by the `<b>` tag is somehow being overridden by a parent

File: src/reducers/proposals.js [hunk 2]
```diff
@@ -130,6 +130,29 @@ function extractProposals(topics, network) {
     }));
     return acc.concat(proposals);
   }, []);
+  return Promise.all(
+    proposals.map(async p => {
+      if (!p.govVote) {
+        try {
+          const [eta, executed] = await Promise.all([
+            window.maker.service('spell').getEta(p.source),
```
Comment (jparklev) [inline src/reducers/proposals.js]: note that this guy now throws if `eta` isn't set. I think that's fine, right? the error will be caught by the catch block and the keys below will stay `undefined`

**Comment by tyler17** [comment]
I realized this caused some tests to fail in test/reducers/proposals.test.js so am working on fixing that first

**Pull request merged**